lib/presentation/one_day_calendar/widgets/calendar_event_tile.dart

```diff
@@ -51,11 +54,14 @@ class CalendarEventTile extends StatelessWidget {
               children: [
                 if (calculatedRowWidth > minWidth)
                   SizedBox(width: calendarSettings.iconSpacingFromText),
-                CalendarTileImage(
-                    event: event,
-                    size: calendarSettings.tileIconSize,
-                    iconBackgroundOpacity:
-                        calendarSettings.iconBackgroundOpacity),
+                if (eventWidth == null
+                    ? true
+                    : eventWidth > calendarSettings.tileIconSize)
```

Comment:
Maybe that way?
`if(eventWidth == null || eventWidth > calendarSettings.titleIconSize)`